app/assets/javascripts/application/sticky_tools.js

```diff
@@ -55,6 +57,3 @@ $(function() {
     scroller();
   }
 });
-
```

Comment:
This is being used to make the save/view/preview/csv buttons sticky on edit action pages (using `data-spy="affix"`). We should probably include it somewhere better, or I'd be open to getting rid of that functionality - just wanted to flag that it would be going away. 

I tried to use it for the actual sticky tools so they could share one solution, but I think the sticky tools need something fancier.